### PR TITLE
[MRG] Remove channel renaming & channel type settings; support latest MNE-BIDS

### PR DESCRIPTION
**Pull request by hoechenberger**
Channel names and types should all be handled via BIDS!

### CODE REVIEW

File: 01-import_and_maxfilter.py [hunk 1]
```diff
@@ -44,7 +43,8 @@
 from mne.parallel import parallel_func
 from mne_bids import make_bids_basename, read_raw_bids, get_matched_empty_room
 from mne_bids.config import BIDS_VERSION
-from mne_bids.utils import _write_json, _parse_bids_filename
+from mne_bids.utils import _write_json
+from mne_bids.path import _parse_bids_filename
```
Comment (jasmainak) [inline 01-import_and_maxfilter.py:47]: do any of these functions need to be made public?

Comment (hoechenberger) [reply 01-import_and_maxfilter.py:47]: We should consider making `_parse_bids_filename` public, yes

**Comment by hoechenberger** [comment]
@jasmainak @agramfort feel free to merge :)

**Comment by agramfort** [comment]
Thx @hoechenberger

**Pull request merged**